.github/workflows/actions.yml

```diff
@@ -34,6 +24,36 @@ jobs:
         with:
           options: --check --diff
           src: .
+      - name: Install flake8, isort
+        run: pip install flake8 isort
+      - name: Lint (flake8)
+        run: flake8
+      - name: Lint (isort)
+        run: isort --check .
```

Comment:
I took liberty to tweak this part a bit so that the flake8 and isort results come back faster than before. (the previous `Install dependencies` step took quite a while and did set up way more things than needed for `flake8` and `isort`)

Reply:
We may want to pin flake8 and isort versions in the future (didn't do so far via requirements-test.txt).